tests/Providers/CloudWatchServiceProviderTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Tests\Providers;
+
+
+use Illuminate\Contracts\Config\Repository;
+use Illuminate\Contracts\Foundation\Application;
+use Monolog\Formatter\JsonFormatter;
+use Monolog\Formatter\LineFormatter;
+use Monolog\Logger;
+use Pagevamp\Providers\CloudWatchServiceProvider;
+
+class CloudWatchServiceProviderTest extends \PHPUnit\Framework\TestCase
+{
+    public function testGetLoggerShouldResolveCustomFormatterInstanceFromConfiguration()
+    {
+        $cloudwatchConfigs = [
+            'name' => '',
+            'region' => '',
+            'credentials' => [
+                'key' => '',
+                'secret' => '',
+            ],
+            'stream_name' => 'laravel_app',
+            'retention' => 14,
+            'group_name' => 'laravel_app',
+            'version' => 'latest',
+            'formatter' => JsonFormatter::class
+        ];
+
+        $config = \Mockery::mock(Repository::class);
+        $config->shouldReceive('get')
+            ->once()
+            ->with('logging.channels')
+            ->andReturn([
+                'cloudwatch' => $cloudwatchConfigs
+            ]);
+        $config->shouldReceive('get')
+            ->once()
+            ->with('logging.channels.cloudwatch')
+            ->andReturn($cloudwatchConfigs);
+
+        $formatter = \Mockery::mock(JsonFormatter::class);
+
+        $app = \Mockery::mock(Application::class);
+        $app->shouldReceive('make')
+            ->once()
+            ->with('config')
+            ->andReturn($config);
+        $app->shouldReceive('make')
+            ->once()
+            ->with(JsonFormatter::class)
+            ->andReturn($formatter);
+
+        $provider = new CloudWatchServiceProvider($app);
+        $logger = $provider->getLogger();
+
+        $this->assertInstanceOf(Logger::class, $logger);
+        $this->assertNotEmpty($logger->getHandlers());
+        $this->assertInstanceOf(
+            JsonFormatter::class,
+            $logger->getHandlers()[0]->getFormatter()
+        );
+    }
+
+    public function testGetLoggerShouldResolveDefaultFormatterInstance()
+    {
+        $cloudwatchConfigs = [
+            'name' => '',
+            'region' => '',
+            'credentials' => [
+                'key' => '',
+                'secret' => '',
+            ],
+            'stream_name' => 'laravel_app',
+            'retention' => 14,
+            'group_name' => 'laravel_app',
+            'version' => 'latest',
+            'formatter' => null
```

Comment:
this is passing because yo have `null` as default value here. This most likely will break if the key is missing. I have added a suggestion to fix this on comment above.